src/AltinnCore/Common/Languages/ini/en.ini

```diff
@@ -89,6 +89,11 @@ information_third_party_library = This component is developed by a third party,
 information_more_info_link = Read more about experiences and where this component is used
 list_all = ALL
 list_favourites = FAVOURITES
+modal_address = Address
+modal_address_zip_code = Zip Code
+modal_address_post_place = Post Place
+modal_address_care_of = C/O or other additional address
+modal_address_house_number = House Number
```

Comment:
I have 1 text related to this advanced component, maybe have the same naming-prefixes as that one.

Reply:
Agreed, fixed.